cnetstat.go

```diff
@@ -116,30 +169,39 @@ func (kc KubeConnection) Fields() []string {
 	}
 }
 
-// Parse our arguments. Return the value of the format argument - either "table" or "json"
-func parseArgs() (string, error) {
-	var format = flag.String("format", "table", "Output format. Either 'table' or 'json'")
+// CnetstatConfig holds our command-line arguments
+type CnetstatConfig struct {
+	outputFormat string // Either "table" or "json"
```

Comment:
Should this be an enum?

Reply:
Thanks! I looked it up, and I did what I think is the idiomatic Go way of handling enums, which is this:
```
type Format int
const (
    tableFormat Format iota,
    jsonFormat
)
```